tests/hello/assembly/main.near.ts

```diff
@@ -12,7 +11,64 @@
       @external("env", "input_read_into")
       declare function input_read_into(ptr: usize): void;
     
-import {contractContext as contractContext,globalStorage as globalStorage,near as near} from "./near";
+import "allocator/arena";
```

Comment:
this means old assemblyscript complier is used. Remove `node_modules` and do `nom install` again, then rebuild